src/fabric/src/fabric_doc_update.erl

```diff
@@ -58,28 +72,34 @@ go(DbName, AllDocs0, Opts) ->
         rexi_monitor:stop(RexiMon)
     end.
 
-handle_message({rexi_DOWN, _, {_, NodeRef}, _}, _Worker, Acc0) ->
```

Comment:
Should not the code still be able to handle messages with the tuple-shaped accumulators?  Such messages can arrive during upgrade where the cluster has mixed version of nodes.

Reply:
the acc is local, the first clause in handle_message is the message term from other nodes